pkg/adapter/kubernetes/blueprintcr/v1/configDiff.go

```diff
@@ -6,7 +6,8 @@ const (
 	ConfigActionNone         ConfigAction = "none"
 	ConfigActionSet          ConfigAction = "set"
 	ConfigActionSetToEncrypt ConfigAction = "setToEncrypt"
-	ConfigActionRemove       ConfigAction = "remove"
+	//TODO: add new actions
```

Comment:
Do you need to add the new action `ConfigActionSetEncrypted` here?

Reply:
I am not sure. This is code from @jelemux from develop. There were a lot of compile errors because of it (i roughly fixed them)  and i think i we should fix further problems in a separate branch.